R/theme-defaults.R

```diff
@@ -154,14 +156,14 @@ theme_grey <- function(base_size = 11, base_family = "",
     axis.line =          element_blank(),
     axis.line.x =        NULL,
     axis.line.y =        NULL,
-    axis.text =          element_text(size = rel(0.8), colour = "grey30"),
+    axis.text =          element_text(size = rel(0.8), colour = col_mix(ink, paper, 0.305)),
```

Comment:
I assume there is a numerical reason why you need the third digit in some but not all of these mix operations?

Reply:
Yeah this is just to land on the right hex code to preserve visual tests. I have no scruples to just take the prettier number here (like 0.3).